main.c

```diff
@@ -51,6 +57,11 @@ void usage(int error)
 	fprintf(dest, "        -s               : Signed using bootsig.bin\n");
 	fprintf(dest, "        -0/1/2/3/4/5/6   : Only look for CMs attached to USB port number 0-6\n");
 	fprintf(dest, "        -h               : This help\n");
+	fprintf(dest, "        -h               : This help\n");
```

Comment:
Duplicate line.

Reply:
done - also moved the recovery text into the main body of the help